automation/update_package_swift.py

```diff
@@ -0,0 +1,77 @@
+import os
+from github import Github
+import hashlib
+import requests
+
+GITHUB_REPO = "mozilla/application-services"
+XC_FRAMEWORK_NAME = "MozillaRustComponents.xcframework.zip"
+PACKAGE_SWIFT = "Package.swift"
+github_access_token = os.getenv("GITHUB_TOKEN")
+as_version = os.getenv("AS_VERSION")
+
+def get_xcframework_artifact():
+    g = Github(github_access_token)
+    repo = g.get_repo(GITHUB_REPO)
+    
+    release = repo.get_release(as_version)
+    for asset in release.get_assets():
+        if asset.name == XC_FRAMEWORK_NAME:
+            return asset.browser_download_url
+
+def compute_checksum(xc_framework_url):
+    req = requests.get(xc_framework_url)
+    return hashlib.sha256(req.content).hexdigest()
```

Comment:
nit: Might be better to chunk up the `content` and feed it into `sha256` in chunks there is support for that in both APIs, and if for some reason (which I don't think we'll ever hit) the `content` is so large it can't fit in memory... at the very minimum I'll make sure this can't reasonably happen

Reply:
I think we do this in a followup if we ever see it (I doubt it)